### PR TITLE
PR 15 test separated into different describes

### PR DESCRIPTION
**Pull request by TakumiMay**

### CODE REVIEW

File: test/buy-tshirt.spec.ts [hunk 1]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
   const signInStepPage: SignInStepPage = new SignInStepPage();
   const summaryStepPage: SummaryStepPage = new SummaryStepPage();
 
-  it('then should be bought a t-shirt', async () => {
+  describe('Open the page in the browser', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: We can use a BDD sintax, here, you can change this by: "Given an online store"

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
...
     await browser.waitForAngularEnabled(false);
     await browser.get('http://automationpractice.com/');
+  });
+
+  describe('T-shirt purchase process', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: "When a user wants to buy a T-Shirt"

File: test/buy-tshirt.spec.ts [hunk 3]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
...
     await menuContentPage.goToTShirtMenu();
     await productListPage.selectFirstItem();
     await productAddedModalPage.proceedToCheckout();
     await summaryStepPage.proceedToCheckout();
+  });
+
+  describe('Sign in to the app', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: "And log-ins to his/her account"

File: test/buy-tshirt.spec.ts [hunk 4]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
...
     await signInStepPage.login('aperdomobo@gmail.com', 'WorkshopProtractor');
-    await addressStepPage.proceedToCheckout();
+  });
 
+  describe('Select the default address', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: "Selects the default address"

File: test/buy-tshirt.spec.ts [hunk 5]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
...
+    await addressStepPage.proceedToCheckout();
     await shippingStepPage.acceptAndContinue();
+  });
 
+  describe('Payment at the bank', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: "And chose the bankwire payment type"

File: test/buy-tshirt.spec.ts [hunk 6]
```diff
@@ -24,21 +24,34 @@ describe('Buy a t-shirt', () => {
...
     await paymentStepPage.payByBankWire();
     await bankPaymentPage.confirmOrder();
 
-    await expect(orderSummaryPage.getOrderTitle()).toBe('Your order on My Store is complete.');
+    it('then should be bought a t-shirt', async () => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: "Then T-Shirt should be bought successfully"

**Review by aperdomob — APPROVED** [review]

**Pull request merged**